src/cli/cli_parser.cc

```diff
@@ -137,3 +138,25 @@ std::string CliParser::ParseMonitor(const std::string& uname) {
 
   return "Monitor complete";
 }
+
+std::string CliParser::ParseStream(const std::string& uname) {
+  if (!(FLAGS_reg.empty() && FLAGS_chirp.empty() && FLAGS_reply.empty() &&
+        FLAGS_read.empty() && FLAGS_follow.empty())) {
+    return "Cannot Register, Reply, Read, or Follow with Monitor.";
+  }
+
+  if (uname.empty()) {
```

Comment:
Do you actually need to be logged in to stream? @barath

Reply:
There isn't a real need, but the way I wrote the spec I think specifies that you need to be logged in for any action.